conclave-host/src/main/resources/gramine-seccomp.json

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
How is this file different to the one in `scripts` directory?

Reply:
No difference. The one in `scripts` is used for integration tests though and somehow I feel like it is part of a separate project.
Not sure how we can reuse that nicely?

Reply:
Something like
```
processResources {
  from "${rootProject.projectDir}/scripts/gramine-seccomp.json"
}
```

in conclave-host/build.gradle.

Reply:
I would have actually done the opposite, i.e. reusing the host resource in the integration test.
But yes, I will got with this solution, thanks!